docs/installation/upgrading.rst

```diff
@@ -1,31 +1,97 @@
 .. _upgrading-ksql:
 
-Upgrading KSQL
-==============
+Upgrading ksqlDB
+================
 
-Upgrade one KSQL server at a time (i.e. rolling restart). The remaining KSQL servers should have sufficient spare
-capacity to take over temporarily for unavailable, restarting servers.
+Upgrade one server at a time in a "rolling restart". The remaining servers
+should have sufficient spare capacity to take over temporarily for unavailable,
+restarting servers.
+
+For general guidance on upgrading, see
+[Upgrade ksqlDB](https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/).
+
+Upgrading to ksqlDB 5.5 from KSQL 5.4
+-------------------------------------
+
+.. important::
+
+   The upgrade from KSQL 5.4 to ksqlDB 5.5 is not a rolling restart. You must
+   shut down all KSQL instances and then start up all ksqlDB instances, so there
+   will be downtime.
+
+#. Prepare a list of SQL statements for migration. One way is to dump the command
+   topic.
+
+   ```bash
+   ./bin/kafka-console-consumer --bootstrap-server localhost:9092 --topic _confluent-ksql-default__command_topic --from-beginning | jq ".statement" 
+   ```
+
+#. List your application's streams, tables, and types. Don't use the EXTENDED
+   option. Go through each list, and find the the last CREATE statement for each
+   stream/table/type name. Order the CREATE statements so they show the correct
+   dependencies.
```

Comment:
Which command are you recommending here? I just used DESCRIBE without EXTENDED. However, there is no CREATE statement in the DESCRIBE command. 

```
ksql> describe ps1;

Name                 : PS1
 Field   | Type                      

 ROWTIME | BIGINT           (system) 
 ROWKEY  | VARCHAR(STRING)  (system) 
 ID      | INTEGER                   

For runtime statistics and query details run: DESCRIBE EXTENDED <Stream,Table>;
```

Should we say the user needs to re-create the CREATE statement based on the displayed schema?

Reply:
From what I've seen, neither DESCRIBE nor DESCRIBE EXTENDED print the CREATE statements for streams/tables, only for persistent queries, in 5.4. In 5.5, DESCRIBE EXTENDED does print CREATE statements for all the streams/tables - which is why I personally first printed all the streams/tables that are present in the system via list; then dumped the topic and found a corresponding CREATE statement for each stream/table so that I could re-create them.

Reply:
@stan-confluent Is it safe to say that you did something like what @big-andy-coates  recommends [here](https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/#how-to-upgrade)? If so, I'll use his procedure.

Reply:
I reworked the steps to be like the procedure here: https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/#how-to-upgrade.